_episodes/02-basics.md

```diff
@@ -272,34 +276,47 @@ Date:   Thu May 4 15:02:56 2017 +0200
 > $ git show   # show various types of objects
 > $ git diff   # show changes
 > ```
-{: .task}
-
-> ## (Optional) Exercise: more changes and renaming
->
-> 1. Apply and commit more changes.
-> 2. Add new files.
-> 3. When you commit, try `git commit -v` (will show you the difference in the editor).
-> 4. Rename files with `git mv` (you will need to `git commit` the rename).
-> 5. Use `git log --oneline` and `git status`.
-{: .task}
+{: .challenge}
 
 > ## (Optional) Exercise: Comparing and showing commits
 >
 > 1. Inspect differences between commit hashes with `git diff <hash1> <hash2>`.
 > 2. Have a look at specific commits with `git show <hash>`.
-{: .task}
+{: .challenge}
+
+> ## (Optional) Exercise: Renaming and removing files
+>
+> 1. Create a new file, `git add` and `git commit` the file.
+> 2. Rename the file with `git mv` (you will need to `git commit` the rename).
+> 3. Use `git log --oneline` and `git status`.
+> 4. Remove the file with `git rm` (again you need to `git commit` the change).
+> 5. Inspect the history with `git log --stat`. Can you recover the removed file from the Git history?
```

Comment:
This is a bigger idea than just commenting on this line (step 5 here), but would it be good pedagogically to give some search engine keywords here as a hint?  Being able to search yourself is really important and it would probably help to nudge people that way.

This is the first place I thought of it but could apply to a whole class of advanced questions.

Reply:
Good idea. I will add this to this PR.